org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NullTestCase.xtend

```diff
@@ -0,0 +1,60 @@
+package org.uqbar.project.wollok.tests.interpreter
+
+import org.junit.Test
+
+class NullTestCase extends AbstractWollokInterpreterTestCase {
+	
+	@Test
+	def void messageSentToNull() {
+		'''
+		assert.throwsException({ null.sayHi() })
```

Comment:
I think that the message should be "receiver of || is null", emphasizing
that the error is the null receiver and not the message sent. Sorry :)

On Thu, Jun 30, 2016 at 10:39 PM, Fernando Dodino <notifications@github.com>
wrote:

> In
> org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NullTestCase.xtend
> <https://github.com/uqbar-project/wollok/pull/772#discussion_r69206172>:
>
> > @@ -0,0 +1,60 @@
> > +package org.uqbar.project.wollok.tests.interpreter
> > +
> > +import org.junit.Test
> > +
> > +class NullTestCase extends AbstractWollokInterpreterTestCase {
> > +	
> > +	@Test
> > +	def void messageSentToNull() {
> > +		'''
> > +		assert.throwsException({ null.sayHi() })
>
> I made a fix, adding a getCause() method in Exception while @Juancete
> <https://github.com/Juancete> is finishing #666
> <https://github.com/uqbar-project/wollok/issues/666>.
> Now you validate "Cannot send message || to null" and other error messages.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/uqbar-project/wollok/pull/772/files/116d2b9ef19f049c8ca980c930013805be426a57#r69206172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEa1ORR25vBBAMJWfrQFNXH2DuTW8pULks5qRCmagaJpZM4I_Wpp>
> .
>